xbmc/cores/VideoPlayer/VideoRenderers/LinuxRendererGLES.h

```diff
@@ -64,13 +64,8 @@ struct YUVCOEF
 
 enum RenderMethod
 {
-  RENDER_GLSL   = 0x001,
-  RENDER_SW     = 0x004,
-  RENDER_POT    = 0x010,
```

Comment:
RENDER_POT is still referenced in CLinuxRendererGLES::CreateYV12Texture(int index) - thats why it doesn't compile ...